ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/recovery/Listener.java

```diff
@@ -242,16 +242,17 @@ public synchronized void stopListener()
            }
            try {
                wait();
-           } catch (InterruptedException e) {
-               // ignore
+           } catch (InterruptedException ignoredIe) {
+               tsLogger.logger.debug("Waiting for connection close was interrupted", ignoredIe);
```

Comment:
It is not ignored anymore so the name needs updating too.

Reply:
I think it's ignored as it's just logged and only under the debug level. But I can rename it.